source/ranges.tex

```diff
@@ -3986,7 +3986,7 @@
 
 \pnum
 Subclause \ref{range.adaptors} defines \term{range adaptors}, which are utilities that transform a
-\libconcept{range} into a \libconcept{view} with custom behaviors. These
+\libconcept{range} into a view with custom behaviors. These
```

Comment:
Do we want `range` in code font here? Doesn't the same principle apply?

Reply:
Yes, I should do that. Thanks for the reminder.